src/components/group-common.js

```diff
@@ -61,15 +62,15 @@ const UserMenuContent = ({ onEdit, onDelete }) => {
 const menuCardSize = 20
 
 export const MemberCardHeaders = Utils.memoWithName('MemberCardHeaders', ({ sort, onSort }) => {
-  return div({ style: { display: 'flex', justifyContent: 'space-between', marginTop: '1.5rem', padding: '0 1rem' } }, [
-    div({ style: { flex: 1 } }, [
+  return div({ role: 'row', style: { display: 'flex', justifyContent: 'space-between', marginTop: '1.5rem', padding: '0 1rem' } }, [
+    div({ role: 'columnheader', 'aria-sort': ariaSort(sort, 'email'), style: { flex: 1 } }, [
       h(HeaderRenderer, { sort, onSort, name: 'email' })
     ]),
-    div({ style: { flex: 1 } }, [
+    div({ role: 'columnheader', 'aria-sort': ariaSort(sort, 'roles'), style: { flex: 1 } }, [
       h(HeaderRenderer, { sort, onSort, name: 'roles' })
     ]),
     // Width is the same as the menu icon.
-    div({ style: { width: menuCardSize } }, [
+    div({ role: 'columnheader', style: { width: menuCardSize } }, [
```

Comment:
just curious - action/buttons in a table don't need a header (like "actions" or "delete")?

Reply:
They do need a header, but it's acceptable for it to be screenreader-only if the column is obvious visually (i.e. action buttons which are all the same). A header won't fit in a really narrow column anyway with looking weird.

`sr-only` makes the header visible to screen readers but hides it visually (unlike `display: none` which hides it from both)